components/ChallengeMaterial/__snapshots__/ChallengeMaterial.test.js.snap

```diff
@@ -623,16 +623,18 @@ exports[`Curriculum challenge page Should render first challenge that is not pas
               class="border border-lightgray rounded my-1"
             >
               <div
-                class="ms-3"
+                class="mx-3"
               >
-                <span
-                  wrapper="code_wrapper"
+                <div
+                  class="my-3"
                 >
-                  A comment under submission
-                </span>
+                  <span>
+                    A comment under submission
+                  </span>
+                </div>
               </div>
               <div
-                class="wrapper d-flex justify-content-between align-items-center p-1"
+                class="wrapper d-flex justify-content-between align-items-center p-2 rounded"
```

Comment:
This was added because the corner feels cutoff as the parent has a rounded border but the child doesn't, so it cuts the parent borders as you can see in the following screenshot:

<img width="50" alt="image" src="https://user-images.githubusercontent.com/35906419/232237598-665add2e-467c-476e-b13b-5473bd07d421.png">

After:

<img width="49" alt="image" src="https://user-images.githubusercontent.com/35906419/232237645-4648ba85-edc6-40ba-b34e-f0226a87d034.png">